app/models/website.rb

```diff
@@ -26,6 +26,22 @@ def website_dependent
     website_members.each(&:delete)
   end
 
+  def origins=(origins, string_origins: "", protocols: ["http", "https", "ftp", "ftps", "sftp"])
```

Comment:
don't do this it's not needed. You need a `before_validation :check_origins`. And in that check_origins method add errors with `errors.add(:origins, 'Please enter a valid origin')` that way when we update and save the website with new origins this error will show up.